openedx/openedx-events
# feat: Add Content Library Collections signals [FC-0062]

**Description:** This PR adds new signals for Content Library Collections.

The following new signals will be added:
- `LIBRARY_COLLECTION_CREATED`  - emitted when new library collection is created
- `LIBRARY_COLLECTION_UPDATED` - emitted when a library collection is updated
- `LIBRARY_COLLECTION_DELETED` - emitted when a library collection is deleted

**ISSUE:** https://github.com/openedx/modular-learning/issues/226

**Dependencies:** Needed for https://github.com/openedx/edx-platform/pull/35321

**Merge deadline:** List merge deadline (if any)

**Installation instructions:** List any non-trivial installation
instructions.

**Testing instructions:**

Make sure tests are passing, and follow testing instructions in https://github.com/openedx/edx-platform/pull/35321

**Reviewers:**

**Merge checklist:**

**Post merge:**
      finished.

**Author concerns:** List any concerns about this PR - inelegant
solutions, hacks, quick-and-dirty implementations, concerns about
migrations, etc.
